ui/app/AppLayouts/Chat/panels/communities/HidePermissionPanel.qml

```diff
@@ -0,0 +1,50 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+
+import StatusQ.Core 0.1
+import StatusQ.Components 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core.Theme 0.1
+
+//import utils 1.0
+
+Control {
+    id: root
+
+    property alias checked: switchItem.checked
+
+    signal toggled
+
+    contentItem: RowLayout {
+        spacing: 16
+
+        StatusRoundIcon {
+            asset.name: "hide"
+        }
+        ColumnLayout {
+            Layout.fillWidth: true
+            StatusBaseText {
+                text: qsTr("Hide permission")
+                color: Theme.palette.directColor1
+                font.pixelSize: 15
+            }
+            StatusBaseText {
+                Layout.fillWidth: true
+                Layout.fillHeight: true
+                text: qsTr("Make this permission hidden from members who don’t meet it’s requirements")
```

Comment:
"its" (typo)

Reply:
Good point, it's also in design (I'm copying it, not rewriting ofc :D), so I fixed here left a comment in figma